tests/test_portal.py

```diff
@@ -168,21 +171,25 @@ def test_message_sent(self):
 
         # confirm styling unicode functions
         body = message.style_message(message.body)
-        self.assertTrue(u'DOCTYPE' in body)
-        self.assertTrue(u'style' in body)
-        self.assertTrue(isinstance(body, unicode))
+        assert u'DOCTYPE' in body
```

Comment:
Please remove the unicode literal marks, they shouldn't be necessary since you imported `unicode_literals`